homeassistant/components/squeezebox/config_flow.py

```diff
@@ -222,3 +242,63 @@ async def async_step_dhcp(
 
         # if the player is unknown, then we likely need to configure its server
         return await self.async_step_user()
+
+
+class OptionsFlowHandler(OptionsFlow):
+    """Options Flow Handler."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Options Flow Steps."""
+
+        errors: dict[str, str] = {}
+
+        async def _validate_input(data: dict[str, Any]) -> str | None:
+            """Validate the user input allows us to connect."""
+
+            if (
+                not isinstance(data[CONF_BROWSE_LIMIT], int)
+                or not data[CONF_BROWSE_LIMIT] > 1
+            ):
+                raise ValueError
+            if (
+                not isinstance(data[CONF_VOLUME_STEP], int)
+                or not data[CONF_VOLUME_STEP] > 1
```

Comment:
```suggestion
                or not data[CONF_VOLUME_STEP] > 0
```
1 should be an acceptable value, right?